utils/templates/latex_preamble.jinja2.md

```diff
@@ -17,12 +17,4 @@ monofontoptions:
 - BoldFont=RobotoMono-Bold.ttf
 - ItalicFont=RobotoMono-Italic.ttf
 - BoldItalicFont=RobotoMono-BoldItalic.ttf
-header-includes:
```

Comment:
removing these is obviously subpar.  At a loss as to what would make things work though.  Any ideas?